src/coralogix.js

```diff
@@ -104,7 +104,19 @@ class CoralogixLogger extends FormattedLoggerBase {
     });
   }
 
-  _logImpl(payload, fields) {
+  async _logImpl(payload, fields) {
+    for (const t of [5, 10, 15]) {
+      try {
+        return this._sendRequest(payload, fields);
+      } catch (e) {
+        await new Promise((res) => setTimeout(res, t));
```

Comment:
`setTimeout` is in **ms**. do you really want to probe that fast?

Reply:
Yes, this leads to a timeout of max <20ms; the outages this is guarded to defend against usually last even shorter. We are already running the danger of OOM/DOSing our selves here in case of a real outage which is why I intentionally chose a very short grace period.

Reply:
Note that the correct solution here would be using bulk transmission; that method has it's own drawbacks (error isolation); this too has solutions (and coralogix has offered to implement a bulk transmission protocol with excellent performance, latency and error isolation)…

But since it looks very much like we are moving the coralogix integration into adobe IO,  I have decided not to spend extra time on this here…